classes/class-path.php

```diff
@@ -82,7 +84,7 @@ public function get_path() {
 		// Ensure the backup directory is protected
 		$this->protect_path();
 
-		return Backup::conform_dir( $this->path );
```

Comment:
Confused as to why you removed the `conform_dir` call here? Isn't this exactly what we want it to do?